src/Aspire.Hosting/ApplicationModel/CustomResourceSnapshot.cs

```diff
@@ -86,6 +95,17 @@ public sealed record CustomResourceSnapshot
     /// The commands available in the dashboard for this resource.
     /// </summary>
     public ImmutableArray<ResourceCommandSnapshot> Commands { get; init; } = [];
+
+    private static HealthStatus? ComputeHealthStatus(ImmutableArray<HealthReportSnapshot> healthReports, string? state)
+    {
+        if (healthReports.Length == 0 && state == "Running")
```

Comment:
Same as previous comment. Also, there should be a known state constant for "Running"